react/features/virtual-backgrounds/reducer.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import { ReducerRegistry } from '../base/redux';
+
+import { BACKGROUND_ENABLED, BACKGROUND_DISABLED } from './actionTypes';
+
+
+ReducerRegistry.register('features/virtual-backgrounds', (state = {}, action) => {
+
+    switch (action.type) {
+    case BACKGROUND_ENABLED: {
```

Comment:
Why is this not part of the virtual-background (no S) feature? Please put all of this in that one feature, why would we want 2, with such similar names even?